contracts/misc/MarketView.sol

```diff
@@ -289,9 +297,13 @@ contract MarketView {
         uint256 _numberOfQuestions = numberOfQuestions(market);
         for (uint256 i = 0; i < _numberOfQuestions; i++) {
             bytes32 questionId = market.questionIDs(i);
-            uint32 finalizeTs = realitio.getFinalizeTS(questionId);
-            if (finalizeTs > 0 && predictions[i] == realitio.getBestAnswer(questionId)) {
-                totalPoints += 1;
+
+            try realitio.resultForOnceSettled(questionId) returns (bytes32 result) {
```

Comment:
same case here, we need to also count partial points, not only once the questions have been settled